docs/_documentations/remote-deploying-codewind.md

```diff
@@ -72,22 +72,28 @@ devex001   codewind    122m   https://codewind-keycloak-devex001.<ingress-domain
 
 ## 3. Add a new user to Keycloak
 
-1\. Copy the **Access URL** returned in the previous step and paste it into a browser.  
+1. Copy the **Access URL** and paste it into a browser.
 
-2\. Ensure that the Realm is set to `Codewind` by clicking on the dropdown arrow on the page. Select **Codewind** if necessary, then:
+2. Click **Administration Console** and log in to Keycloak with `admin` as the default login user name and `admin` as the default password.
 
+3. Change the administrator password and return to the **Administration Console**.
+- Click the **Admin** link and choose **Manage Account/Password**.
+- Set a new administrator password.
+- To return to the admin console, use the **Admin** link or log out and log back in to Keycloak as the admin user with your new admin password.
```

Comment:
Actually, the link to return to the admin console is: 
![image](https://user-images.githubusercontent.com/9484320/79979427-7fe26080-846f-11ea-8d5f-1a5d7e8ee118.png)